totalRP3/core/impl/slash.lua

```diff
@@ -78,6 +99,58 @@ function SlashCmdList.TOTALRP3(msg, editbox)
 	end
 end
 
+local function testConditions(conditionals)
+	logger:Info(Ellyb.Tables.toString(conditionals));
+	local conditionalsPassed = false;
+	for _, condition in pairs(conditionals) do

```

Comment:
Is `testConditions` called with an _ordered_ table of conditionals? If I'm reading rightly due to `pairs` here things wouldn't work so well if I had a bunch of conditionals in a macro since they'd potentially get iterated in an indeterminate order.